crypto/store/store_lib.c

```diff
@@ -935,6 +935,11 @@ OSSL_STORE_CTX *OSSL_STORE_attach(BIO *bp, const char *scheme,
     if ((loader = ossl_store_get0_loader_int(scheme)) != NULL)
         loader_ctx = loader->attach(loader, bp, libctx, propq,
                                     ui_method, ui_data);
+    // ossl_store_get0_loader_int will will raise an error if the loader for
+    // the scheme cannot be retrieved. This error is cleared as the code below
+    // will try to fetch the loader from the provider.
+    if (loader == NULL)
+      ERR_clear_error();
```

Comment:
`ERR_set_mark()` and `ERR_pop_to_mark()` would be a better way I think.

Reply:
I'll take a look at using them. Thanks